include/votca/xtp/internalcoords.h

```diff
@@ -0,0 +1,78 @@
+#ifndef _VOTCA_XTP_INTERNAL_COORDS_H
+#define _VOTCA_XTP_INTERNAL_COORDS_H
+#include <boost/graph/adjacency_list.hpp>
+#include <iostream>
+#include <vector>
+#include <votca/tools/elements.h>
+#include <votca/xtp/CoordContainer.h>
+#include <votca/xtp/cartesiancoords.h>
+#include <votca/xtp/coordbase.h>
+#include <votca/xtp/eigen.h>
+#include <votca/xtp/orbitals.h>
+#include <votca/xtp/qmatom.h>
+
+namespace votca {
+namespace xtp {
+
+// Bonds will be stored in an undirected graph
+typedef boost::adjacency_list<boost::vecS, boost::vecS, boost::undirectedS>
+    BglGraph;
+
+class InternalCoords : public CoordBase {
```

Comment:
This is essentially a graph right? why not call it InternalGraph? InternalCoords does not imply anything to do with bond structure.